backend/simulation/realm/truck.py

```diff
@@ -42,6 +42,13 @@ def destination(self) -> int:
             return -1
         return self._route[-1]
 
+    @property
+    def next_container_id(self) -> int:
+        if len(self._route) == 0:
+            return -1
+        else:
+            return self._route[0]
+
```

Comment:
This is currently incorrect since at the moment the route is stored in reverse order... I think we're changing that so just making a note that we need to change that before merging this.